facdb/bash/dataloading.sh

```diff
@@ -3,7 +3,8 @@ CURRENT_DIR=$(dirname "$(readlink -f "$0")")
 source $CURRENT_DIR/config.sh
 max_bg_procs 5
 
-import_public dcp_facilities &
+import_public dcp_facilities_with_unmapped $VERSION_PREV &
```

Comment:
Naming is one of the hardest things that we do.  I don't love `dcp_facilities_with_unmapped` as a name, but I'm not convinced my ideas are better (i.e. `dcp_facilities_csv` or `dcp_facilities_full`)

Reply:
I actually like the explicity of the `dcp_facilities_with_unmapped` if too long we might shorten it to  `dcp_facilities_wunmap`?

Reply:
I'm open to however we want to change it. I think the `_csv` is confusing as the data can be in data library as any format. And I think `_full` is less clear as to what records the other dataset is missing.

Reply:
I don't think there is an issue with a long name  as long as it is descriptive